src/main/java/com/yelp/nrtsearch/server/luceneserver/search/query/multifunction/MultiFunctionScoreQuery.java

```diff
@@ -262,6 +280,80 @@ public boolean isCacheable(LeafReaderContext ctx) {
     }
   }
 
+  public static class MinScoreWrapper extends Scorer {
```

Comment:
The file linked in the PR text uses the elastic license. Do not copy code from this file. If you need to include code based on elasticsearch, ensure you are using a version under the apache 2.0 license and cite the source in the class header comment.